### PR TITLE
GG-25549 .NET: Add NuGet verification script

### PR DESCRIPTION
**Pull request by ptupitsyn**
Add a script to verify NuGet packages:
* Creates a test project with `dotnet new console`
* Installs all packages from specified folder with `dotnet add package`
* Runs a test program that uses basic Cache, SQL and LINQ APIs
* Checks exit codes to make sure there were no exceptions/errors

### CODE REVIEW

File: modules/platforms/dotnet/release/verify-nuget.ps1 [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+#>
+
+param (
+    [string]$packageDir="..\nupkg"
+)
+
+
+# Find NuGet packages (*.nupkg)
+$dir = Join-Path $PSScriptRoot $packageDir
```
Comment (isapego) [inline modules/platforms/dotnet/release/verify-nuget.ps1]: Does it only work with relative paths?

File: modules/platforms/dotnet/release/Program.cs [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            }
+        }
+    }
+
+    public class Person
+    {
+        public Person(int age)
+        {
+            Age = age;
+            Name = $"Person-{age}";
```
Comment (isapego) [inline modules/platforms/dotnet/release/Program.cs]: Wow, didn't know .NET can do that.

Comment (ptupitsyn) [reply modules/platforms/dotnet/release/Program.cs]: Killer feature, right? 😁 
There is lots of impressive stuff in recent versions, but we are stuck with 4.0 in the main code base for now :(
This script uses modern tooling (.NET Core), so we can use newer language features.

File: modules/platforms/dotnet/release/verify-nuget.ps1 (hunk 1 above)
Comment (ptupitsyn) [reply modules/platforms/dotnet/release/verify-nuget.ps1]: Good point, fixed

**Pull request merged**